test/utils/helpers.ts

```diff
@@ -1,26 +1,92 @@
 import { BigNumber } from 'ethers';
-import { ethers } from 'hardhat'  
+import { ethers } from 'hardhat';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 
-
-export const zero = ethers.constants.Zero; 
+export const zero = ethers.constants.Zero;
 
 export interface Reward {
   _address: string;
   _amount: BigNumber;
+  _signer?: SignerWithAddress;
 }
 
 /**
- * @notice Sums rewards amounts that are due to voters 
- * @param rewards Array of structs of voters' addresses and amounts 
- * to be transfered 
+ * @notice Sums rewards amounts that are due to voters
+ * @param rewards Array of structs of voters' addresses and amounts
+ * to be transfered
  */
 
 export function sum(rewards: Reward[]): BigNumber {
-  let sum = zero;
+  let SUM = zero;
```

Comment:
I'd keep the previous casing. This makes it look like a constant, which it's not

Reply:
Ok - I updated the name of the function to getSum